### PR TITLE
MOBILEAPPS-2052

### PR DESCRIPTION
**Pull request by aman-alfresco**
- added start process on the preview screen

### CODE REVIEW

File: actions/src/main/res/layout/view_list_process_message.xml
Comment (ankitgoyalgloballogic) [inline actions/src/main/res/layout/view_list_process_message.xml:1]: Are we creating new files for workflows?

File: actions/src/main/res/values-de/strings.xml
Comment (ankitgoyalgloballogic) [inline actions/src/main/res/values-de/strings.xml:1]: Are there any changes in the localization? Why did we update this?

**Review by ankitgoyalgloballogic — CHANGES_REQUESTED** [review]

File: actions/src/main/res/layout/view_list_process_message.xml
Comment (aman-alfresco) [reply actions/src/main/res/layout/view_list_process_message.xml:1]: we are not creating the new files. I just optimized the code to enable the process functionality on the preview screen

File: actions/src/main/res/values-de/strings.xml
Comment (aman-alfresco) [reply actions/src/main/res/values-de/strings.xml:1]: moved the string from the listview to the actions module. Now these strings are available for more modules for common use.

**Review by ankitgoyalgloballogic — APPROVED** [review]

**Pull request merged**